dqc_us_rules/dqc_us_0006.py

```diff
@@ -75,6 +75,20 @@ def _dict_list_update(dict_a, dict_b):
         dict_a[key].extend(val)
     return dict_a
 
+def date_bounds_from_csv():
```

Comment:
might consider marking this as a private function by prefixing the function name with an underscore: ```_date_bounds_from_csv```